ysql_test.go

```diff
@@ -440,6 +440,7 @@ func TestUpdateUser_Expiration(t *testing.T) {
 	// Shared test container for speed - there should not be any overlap between the tests
 	db, cleanup := getysql(t, nil)
 	defer cleanup()
+	db.db.SetMaxOpenConns(1)
```

Comment:
Add this as well as how users can retry to avoid catalog mismatch errors in the documentation as best practices or something.